lib/requestwrapper.ts

```diff
@@ -270,14 +270,20 @@ export class RequestWrapper {
       if (isAuthenticationError(axiosError)) {
         error.message = 'Access is denied due to invalid credentials.';
       }
-
     } else if (axiosError.request) {
       // The request was made but no response was received
       // `error.request` is an instance of XMLHttpRequest in the browser and an instance of
       // http.ClientRequest in node.js
-      error.message = 'Response not received. Body of error is HTTP ClientRequest object';
-      error.body = axiosError.request;
-
+      error.message = axiosError.message;
+      error.statusText = axiosError.code;
+      error.body = 'Response not received - no connection was made to the service.';
+
+      // when a request to a private cloud instance has an ssl problem, it never connects and follows this branch of the error handling
+      if (isSelfSignedCertificateError(axiosError)) {
+        error.message = `If you're trying to call a service on ICP or Cloud Pak for Data, you ` +
+          `may not have a valid SSL certificate. If you need to access the service without setting that up, try using ` +
```

Comment:
What about?

> The service you are trying to call doesn't use valid certificates. You can disable the SSL certificate validation using the `disableSslVerification` option.

Reply:
I'm fine with that. I just copied the message that @lpatino10 has in Java for consistency. Would you rather this change?